src/docs/Contributing/InstallAndRun/index.mdx

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
This file was deleted in a previous PR. So I think you can delete it on your branch.

Reply:
Indeed!